docs/topics/manually-test-rules.adoc

```diff
@@ -9,7 +9,7 @@
 You can run an XML rule against your application file to test it:
 
 ----
-$ <MTA_HOME>/bin/mta-cli [--sourceMode] --input <INPUT_ARCHIVE_OR_FOLDER> --output <OUTPUT_REPORT_DIRECTORY> --target <TARGET_TECHNOLOGY> --packages <PACKAGE_1> <PACKAGE_2> <PACKAGE_N>
+$ <{ProductShortName}_HOME>/bin/mta-cli [--sourceMode] --input <INPUT_ARCHIVE_OR_FOLDER> --output <OUTPUT_REPORT_DIRECTORY> --target <TARGET_TECHNOLOGY> --packages <PACKAGE_1> <PACKAGE_2> <PACKAGE_N>
```

Comment:
Did QE verify this?

Reply:
??. When in doubt, ask QE and write what they say appears or will appear on 17 November.